CPAC/qc/utils.py

```diff
@@ -1840,9 +1843,9 @@ def make_montage_sagittal(overlay, underlay, png_name, cbar_name):
     matplotlib.rcParams.update({'font.size': 5})
 
     try:
-        from mpl_toolkits.axes_grid1 import ImageGrid   
+        from mpl_toolkits.axes_grid1 import ImageGrid
     except:
-        from mpl_toolkits.axes_grid import ImageGrid
+        from mpl_toolkits.axes_grid1 import ImageGrid
```

Comment:
This `try-except` is not necessary; just import `from mpl_toolkits.axes_grid1` one time only.

Reply:
Thanks for catching that @pn2200! Resolved in af3e712a